x-pack/plugins/cases/common/types/api/connector/v1.ts

```diff
@@ -6,8 +6,25 @@
  */
 
 import * as rt from 'io-ts';
-import { CaseConnectorRt } from './connector';
-import { CaseExternalServiceBasicRt } from '../cases';
+import { CaseExternalServiceBasicRt } from '../../../api';
+import { CaseConnectorRt } from '../../domain/connector/v1';
+
+export * from './mappings.v1';
```

Comment:
Does this need to be a different file?

Reply:
No. Do you prefer to have it in `x-pack/plugins/cases/common/types/api/connector/v1.ts`? Tbh, I struggled a lot with what to do with this 🙂

Reply:
Yeah I think it would minimize the duplicated exports and make things more linear. It is a nit though so feel free to leave it if you prefer as is 🙌

Reply:
I will merge them. Seems to be better.